superduperdb/db/base/db.py

```diff
@@ -531,7 +531,7 @@ def _build_task_workflow(
         for identifier in listener:
             info = self.metadata.get_component('listener', identifier)
             query = info['dict']['select']
-            model, key = identifier.split('/')
+            model, _, key = identifier.rpartition('/')
```

Comment:
I _think_ this is fine. Note that [here](https://github.com/SuperDuperDB/superduperdb/blob/06bf7c1af3d4e3be8d86595f83cd6c31764233b1/superduperdb/container/listener.py#L75-L76) we have a recursive definition of `identifier`. I don't know if assigning `model` to the first tuple value here will cause problems somewhere else because of this recursive definition.

(But the current fix certainly appears more correct than what was there before.)

Reply:
Yes this should work.